src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcConnection.cs

```diff
@@ -223,7 +223,7 @@ internal bool IsV3Driver
                     {
                         try
                         {   // mdac 89269: driver may return malformatted string
-                            ProviderInfo.IsV3Driver = (int.Parse(ProviderInfo.DriverVersion.Substring(0, 2), CultureInfo.InvariantCulture) >= 3);
+                            ProviderInfo.IsV3Driver = (int.Parse(ProviderInfo.DriverVersion.AsSpan(0, 2), CultureInfo.InvariantCulture) >= 3);
```

Comment:
Analyzer surfaced this one, since `int.Parse` has a valid Span overload exposed now.

Reply:
Do we reference the System.Memory package from the netstandard build here?

Reply:
Yes it does. I missed that at the very end of the list.

We'll either need to `#ifdef` or suppress the analyzer, since it was surfacing as an error.

Reply:
We can just suppress that analyzer for the whole csproj.  It's what we've done in other such assemblies where the minor perf gains on non-hot-paths don't warrant ifdef'ing/partial files/etc.

Reply:
Fixed.